src/main/java/org/commonjava/indy/service/repository/jaxrs/RepositoryQueryResources.java

```diff
@@ -152,11 +155,13 @@ public Response getAllByDefaultPackageTypes()
     @Path( "/byName/{name}" )
     @Produces( APPLICATION_JSON )
     public Response getByName(
-            @Parameter( description = "Name of the repository", required = true ) @PathParam( "name" ) String name )
+            @Parameter( description = "Name of the repository", required = true ) @PathParam( "name" ) @Encoded
```

Comment:
It seems that the decoding issue will only happen on QueryParam, maybe we could leave this out.

Reply:
Oh, this is a typo. Will ignore it.